x-pack/test/cases_api_integration/security_and_spaces/tests/common/cases/import_export.ts

```diff
@@ -161,32 +164,22 @@ const expectImportToHaveOneCase = async (supertestService: supertest.SuperTest<s
 
 const expectImportToHaveCreateCaseUserAction = (userAction: CaseUserActionResponse) => {
   expect(userAction.action).to.eql('create');
-  expect(includesAllCreateCaseActionFields(userAction.action_field)).to.eql(true);
 };
 
 const expectImportToHavePushUserAction = (userAction: CaseUserActionResponse) => {
-  expect(userAction.action).to.eql('push-to-service');
-  expect(userAction.action_field).to.eql(['pushed']);
-  expect(userAction.old_value).to.eql(null);
-
-  const parsedPushNewValue = JSON.parse(userAction.new_value!);
-  expect(parsedPushNewValue.connector_name).to.eql('A jira connector');
-  expect(parsedPushNewValue).to.not.have.property('connector_id');
-  expect(userAction.new_val_connector_id).to.eql('1cd34740-06ad-11ec-babc-0b08808e8e01');
+  const pushedUserAction = userAction as PushedUserAction;
+  expect(userAction.action).to.eql('push_to_service');
+  expect(userAction.type).to.eql('pushed');
+
+  expect(pushedUserAction.payload.externalService.connector_name).to.eql('A jira connector');
```

Comment:
Should we check the `connector_id` here too?

Reply:
Yeap!